bayesian_exercises/election_poll/election_poll.jl

```diff
@@ -0,0 +1,176 @@
+### A Pluto.jl notebook ###
+# v0.11.12
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 036183d2-f775-11ea-0976-eb09b0c00f66
+begin
+	using Soss
+	using Distributions
+	using Plots
+end
+
+# ╔═╡ 5e231d96-f778-11ea-3c30-f134d4b4b537
+md"## Using Soss.jl to analize an election poll"
+
+# ╔═╡ 87ccbd50-f778-11ea-3451-97f4184db99d
+md"We have data about the preferences of 1,447 likely voters:
+
+Bush: 727
+Dukakis: 583
+Other: 137
+"
+
+# ╔═╡ f1fcfc00-f778-11ea-322a-95b2444823ff
+begin
+	data = [727,583,137]
+	n = sum(data)
+	k = length(data)
+	p = data ./n
+end;
+
+# ╔═╡ 02721cc8-f781-11ea-0894-67ee2caeea30
+n
+
+# ╔═╡ 25630eb6-f779-11ea-0779-1f36f318865f
+md"Modelating this data as a Multinomial Random Variable, with each probability of success as a Dirichlet distribution.
+
+The dirichlet distribution can be seen as a generalization of the beta distribution. Initializing it as a non-informative uniform, we have:" 
+
+# ╔═╡ d45aeb32-f779-11ea-39cd-dd159722ab98
+m = @model begin
+	a = ones(k)
+	theta ~ Dirichlet(a) 
+	diff_bush_Dukakis = theta[1] - theta[2]
+	
+	x ~ Multinomial(n, theta) 
+end;
+
+# ╔═╡ e6185a68-f77d-11ea-39e3-87e57ebf2022
+post_prob = dynamicHMC(m(k=k,n=n), (x=data,))
+
+# ╔═╡ bdb22550-f781-11ea-1623-437b271a730c
+begin
+	post_bush = [i.theta[1] for i in post_prob]
+	post_dukakis = [i.theta[2] for i in post_prob]
+end;
+
+# ╔═╡ e27b4696-f783-11ea-26ac-010e5dd402a7
+histogram(post_bush, title = "Posterior probability of Bush wining", label = false)
+
+# ╔═╡ 272e63ca-f784-11ea-069a-3fb3c8fe96a9
+histogram(post_dukakis, title = "Posterior probability of Dukakis wining", label = false)
+
+# ╔═╡ 4ff6cfea-f784-11ea-09cd-bfe2241233ee
+post_diff_bush_dukakis = [i.theta[1] - i.theta[2] for i in post_prob]
+
+# ╔═╡ 6c1a7960-f784-11ea-296a-41c248a97b37
+histogram(post_diff_bush_dukakis, title = "Posterior difference between Bush and Dukakis", label = false)
```

Comment:
A short explanation here would be useful. Is Bush more likely to win according to the model? How big is the lead?